tests/conftest.py

```diff
@@ -48,8 +48,8 @@ def route53_session(aws_credentials):
         yield session.client("route53")
 
 
-@pytest.fixture(scope="function")
-def iam_session(aws_credentials):
-    with mock_iam():
-        session = boto3.session.Session(profile_name="foo", region_name="eu-west-2")
-        yield session.client("iam")
\ No newline at end of file
+# @pytest.fixture(scope="function")
+# def iam_session(aws_credentials):
+#     with mock_iam():
+#         session = boto3.session.Session(profile_name="foo", region_name="eu-west-2")
+#         yield session.client("iam")
```

Comment:
Do we not need this anymoar? Could be removed(?).

Reply:
yep removing

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/73/commits/45107de08e0ffe74cbfaa744aa07a5b2cd8db584.